test_unicode test failing on win 3.3

`test_unicode:test_install_package_that_emits_unicode()` fails on windows 3.3

test was added in pull #374 to address issue #326.

skipping the test for now until someone can look at this closer.

```
=================================================================
FAIL: Install a package with a setup.py that emits UTF-8 output and then fails.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\Users\Administrator\marcus\pipVE3\lib\site-packages\nose\case.py", li
ne 198, in runTest
    self.test(*self.arg)
  File "c:\Users\Administrator\marcus\pip\tests\test_unicode.py", line 24, in te
st_install_package_that_emits_unicode
    assert 'FakeError: this package designed to fail on install' in result.stdou
t, str(result)
AssertionError: Script result: pip install c:\Users\Administrator\marcus\pip\tes
ts\packages\BrokenEmitsUTF8
  return code: 1
-- stdout: --------------------
Unpacking c:\users\administrator\marcus\pip\tests\packages\brokenemitsutf8
  Running setup.py egg_info for package from file:///c%7C%5Cusers%5Cadministrato
r%5Cmarcus%5Cpip%5Ctests%5Cpackages%5Cbrokenemitsutf8
    Traceback (most recent call last):
      File "<string>", line 16, in <module>
      File "c:\Users\Administrator\marcus\pip\tests\tests_cache\test_ws\.virtual
env\lib\encodings\cp1252.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 458:
character maps to <undefined>
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):

  File "<string>", line 16, in <module>

  File "c:\Users\Administrator\marcus\pip\tests\tests_cache\test_ws\.virtualenv\
lib\encodings\cp1252.py", line 23, in decode

    return codecs.charmap_decode(input,self.errors,decoding_table)[0]

UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 458: char
acter maps to <undefined>

----------------------------------------
Command python setup.py egg_info failed with error code 1 in c:\users\administra
tor\marcus\pip\tests\tests_cache\test_ws\tmp\pip-lw2nqo-build
Storing complete log in c:\Users\Administrator\marcus\pip\tests\tests_cache\test
_ws\pip-log.txt

-- created: -------------------
  pip-log.txt  (2599 bytes)
  tmp\pip-lw2nqo-build
          broken.py  (0 bytes)
          pip-egg-info
          setup.py  (1043 bytes)

----------------------------------------------------------------------
```

--- a/tests/functional/test_install.py
+++ b/tests/functional/test_install.py
@@ -924,8 +924,6 @@ def test_install_package_conflict_prefix_and_user(script, data):
     )
 
 
-# skip on win/py3 for now, see issue #782
-@pytest.mark.skipif("sys.platform == 'win32' and sys.version_info >= (3,)")
 def test_install_package_that_emits_unicode(script, data):
     """
     Install a package with a setup.py that emits UTF-8 output and then fails.

